src/app/page.js

```diff
@@ -1,7 +1,14 @@
 "use client"
-import Image from 'next/image';
 import { useState } from "react";
 import { useRouter } from 'next/navigation';
+import Image from 'next/image';
+import home from './images/home.png';
+import user from './images/user.png';
```

Comment:
as far as I know, we do not need these things you added, please use this website as your reference don't just come up with something of your knowledge no, follow this website https://javascript.info/ and do the exactly design